frontend/src/components/messages/RequestConversations.vue

```diff
@@ -151,6 +175,9 @@ export default {
     // When assistanceRequestId changes, get the conversation for the new assistance request.
     assistanceRequestId: async function (newVal) {
       this.loading = true
+      //close panel if open, so it re-renders properly
+      this.closePanel()
+      this.loadedAttachments = undefined
```

Comment:
It's usually best to reset to null as opposed to undefined since we are explicitly removing the value.

Reply:
makes sense! done